src/htmlParser.js

```diff
@@ -32,12 +32,16 @@ class HtmlParser {
 
       const contentText = comment.contentText.runs
 
+      const customEmojis = []
       contentText.forEach((content) => {
         if (content.text.trim() === '') {
           text = text + '<br>'
         } else {
           text = text + content.text
         }
+        if (typeof content.emoji !== 'undefined') {
+          customEmojis.push({"text":content.text,"emojiThumbnails":content.emoji.image.thumbnails})
```

Comment:
Why do you use quotes for keys?
Only JSON requires keys to be quoted
Not saying you can't but doesn't that require more typing?